### ISSUE TITLE
Desktop file didn't specify Exec field

### ISSUE THREAD

**Issue by Toyonut** [PowerShell/PowerShell-Snap#59]
Hi

I just tried to launch powershell 7.0 from the gnome desktop applications on an Ubuntu 19.10 based system. I get a popup `Desktop file didn't specify Exec field`.

Looking in the desktop file under `/var/lib/snapd/desktop/applications/powershell_powershell-preview.desktop` I see:
```
[Desktop Entry]
X-SnapInstanceName=powershell
Name=PowerShell Preview
Type=Application
Terminal=true
Categories=ConsoleOnly;System
Icon=/snap/powershell/104/meta/gui/icon.png
```

To fix it I need to add `Exec=/snap/bin/pwsh` like so:

```
[Desktop Entry]
X-SnapInstanceName=powershell
Name=PowerShell Preview
Type=Application
Terminal=true
Categories=ConsoleOnly;System
Icon=/snap/powershell/104/meta/gui/icon.png
Exec=/snap/bin/pwsh
```

I'm not quite sure how I ended up on preview as I had 6.X stable installed, but that isn't the end of the world. The file at https://github.com/PowerShell/PowerShell-Snap/blob/2a34e06443bf690d7eaf0df148cc03108b6b56d3/preview/snap/gui/powershell-preview.desktop#L4 does have an Exec field, so I'm not sure where this is going missing. I'm also not sure if this affects older versions, I don't tend to launch powershell from the applications menu in gnome.

**Comment by Toyonut** [comment]
Just tried to reinstall Powershell via snap with `sudo snap install powershell --classic` and it has come up with the powershell preview install again and the same error about Desktop file didn't specify Exec field

**Comment by TravisEz13** [comment]
This might be better to post on the snapcraft.io forum and mention me.  This is not a field they say to include.

**Comment by Toyonut** [comment]
Hi @TravisEz13. I have just installed a new system with Powershell from snap on Ubuntu 20.04. My last 20.04 install had the same issue as I originally posted. 
I just noticed my Powershell icon is now back to being the old icon and not the preview one and it works again from the icon with no error. 

I can see the .desktop file has 
```
[Desktop Entry]
Name=PowerShell
Exec=powershell
Type=Application
Terminal=true
Categories=ConsoleOnly;System
Icon=${SNAP}/meta/gui/icon.png
```
and also Powershell is now 7.0.1. 

This issue appears to have been resolved with the new stable release. I will close this issue now.
